README.md

```diff
@@ -7,48 +7,55 @@
 
 # Tracee - Container and system tracing using eBPF
 
-Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real-time. A unique feature of Tracee is that it will only trace newly created processes and containers (that were started after Tracee has started), in order to help the user focus on relevant events instead of every single thing that happens on the system (which can be overwhelming). Adding new events to Tracee (especially system calls) is straightforward, and will usually require no more than adding few lines of code.
-
-Other than tracing, Tracee is also capable of capturing files written to disk or memory ("fileless"), and extracting binaries that are dynamically loaded to an application's memory (e.g. when an application uses a packer). With these features, it is possible to quickly gain insights about the running processes that previously required the use of dynamic analysis tools and special knowledge.
+Tracee is a lightweight and easy to use tracing tool for Linux, which is focused on security and forensics.
```

Comment:
I think we need to keep some of the intro, don't you think?

Reply:
I think we either say something general about unique features or don't say anything and let the demo speak for itself. what do you prefer?

Reply:
I think we need here at least one paragraph which describes tracee